src/_includes/docs/debug/debug-flow-android.md

```diff
@@ -0,0 +1,95 @@
+{{site.alert.note}}
+  If you want to use the [GNU Project Debugger][] to debug the
+  Flutter engine running within an Android app process,
+  check out [`flutter_gdb`][].
+{{site.alert.end}}
+
+[GNU Project Debugger]: https://www.sourceware.org/gdb/
+[`flutter_gdb`]: https://github.com/flutter/engine/blob/main/sky/tools/flutter_gdb
+
+#### Build the Android version of the Flutter app in the Terminal
+
+To generate the needed Android platform dependencies,
+run the `flutter build` command.
+
+```terminal
+$ flutter build appbundle --debug
+```
+
+```terminal
+Running Gradle task 'bundleDebug'...                               27.1s
+✓ Built build/app/outputs/bundle/debug/app-debug.aab.
+```
+
+#### Start debugging with VS Code first
+
+If you use VS Code to debug most of your code, start with this section.
```

Comment:
Could we link to the other section here? Perhaps something like:

```suggestion
If you use VS Code to debug most of your code, start with this section.
If you use XCode to debug most of your code, see [Start debugging with XCode first](...)
```

Reply:
Alternatively, could this be tabs?

Reply:
I'm running into an issue with tabs. I'll make your changes then circle back to that.

Reply:
Just found my tab solution. I'll use it here.

Reply:
It looks like the sidebar's outline has `Start from X` links, but they don't work due to the tabs. Should we remove the headers? They seem redundant with the tabs' titles.